src/constants/index.js

```diff
@@ -5,4 +5,89 @@ export const refShapes = [
   PropTypes.shape({ current: PropTypes.instanceOf(Element) }),
 ];
 
-export default { refShapes };
+export const theme = {
+  colors: {
+    brand: {
+      primary: 'var(--f-color-brandPrimary)',
+      primaryLight: 'var(--f-color-brandPrimary--light)',
+      primaryInactive: 'var(--f-color-brandPrimary--inactive)',
+      secondary: 'var(--f-color-brandSecondary)',
+      secondaryLight: 'var(--f-color-brandSecondary--light)',
+    },
+    icon: {
+      white: 'var(--f-color-icon--white)',
+      disabled: 'var(--f-color-icon--disabled)',
+      dark: 'var(--f-color-icon--dark)',
+    },
+    element: {
+      primary: 'var(--f-color-element--primary)',
+      secondary: 'var(--f-color-element--secondary)',
+      tertiary: 'var(--f-color-element--tertiary)',
+      inactive: 'var(--f-color-element--inactive)',
+      error: 'var(--f-color-element--error)',
+      success: 'var(--f-color-element--success)',
+      overlay: 'var(--f-color-element--overlay)',
+    },
+    text: {
+      primary: 'var(--f-color-text--primary)',
+      secondary: 'var(--f-color-text--secondary)',
+      tertiary: 'var(--f-color-text--tertiary)',
+      link: 'var(--f-color-text--link)',
+      white: 'var(--f-color-text--white)',
+    },
+  },
+
+  fontSizes: {
```

Comment:
I think it's more intuitive to use a singular name: `fontSize`, `space` `fontWeight` rather than the plural, same as with the ones above (text, element, icon, etc). Then we use it as `theme.fontSize.xs`, rather than `theme.fontSizes.xs` and so on.

Reply:
Good point 👍